test/apisource.yaml

```diff
@@ -0,0 +1,55 @@
+# https://knative.dev/development/developer/eventing/sources/apiserversource/getting-started/
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: apiserversource
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: apiserversource
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: apiserversource
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: apiserversource
+subjects:
+- kind: ServiceAccount
+  name: apiserversource
+  namespace: event-example
+---
+apiVersion: sources.knative.dev/v1
+kind: ApiServerSource
+metadata:
+ name: apiserversource
+ namespace: event-example
```

Comment:
This namespace doesn't exist so you'll need to add it - unsure which resources should being in this namespace or just `default` (when the namespace is omitted)

Reply:
I assume the end user is following the linked example.

Reply:
I assume the end user is following the linked example. `k create ns event-example`

Reply:
I don't see that in the `test/README.md` file -  are you referring to something else?

Reply:
Added a note to the links